hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -173,6 +174,16 @@ protected KeyArgs resolveBucketAndCheckOpenKeyAcls(KeyArgs keyArgs,
     return resolvedArgs;
   }
 
+  protected void validateKeyName(OzoneManager ozoneManager, String keyName)
```

Comment:
nit: would it be better to pass `keyArgs` here in place of `keyName`?

Reply:
Thanks @hemantk-12 .
The verification key name is different in OMFileCreateRequest, OMKeyCommitRequest, OMKeyCreateRequest, and OMKeyRenameRequest.
`
OMFileCreateRequest, Verify keyName=StringUtils.removeEnd(keyArgs.getKeyName(),    OzoneConsts.FS_FILE_COPYING_TEMP_SUFFIX)
`
`
OMKeyCommitRequest, Verify keyName=StringUtils.removeEnd(keyArgs.getKeyName(),OzoneConsts.FS_FILE_COPYING_TEMP_SUFFIX)
`
`
OMKeyCreateRequest, Verify keyName=keyArgs.getKeyName()
`
`
OMKeyRenameRequest, Verify keyName=keyRequest.getToKeyName()
`
So I extracted the keyName separately as a parameter.

Reply:
If keyArgs is passed as a parameter, is it reclassified inside OMKeyRequest#validateKeyName()?

Reply:
I thought it would be better to pass `keyArgs` in case we needed something else for it later in future.
It is not a big concern as such and can be left as it is for now.